### PR TITLE
PYTHON-3011 Skip test_connections_are_only_returned_once on PyPy for now

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

**Review by NoahStapp — APPROVED** [review]
LGTM, although the title of the PR is confusing since this change disables that test on PyPy.

**Comment by blink1073** [comment]
> although the title of the PR is confusing since this change disables that test on PyPy.

Missed a word, thanks!

File: test/test_load_balancer.py [hunk 1]
```diff
@@ -37,6 +37,8 @@ class TestLB(IntegrationTest):
     RUN_ON_SERVERLESS = True
 
     def test_connections_are_only_returned_once(self):
+        if "PyPy" in sys.version:
+            self.skipTest("Test is flaky on PyPy")
```
Comment (ShaneHarvey) [inline test/test_load_balancer.py:41]: Can you add the jira ticket to the skip message? It's helpful for tracking

File: test/test_load_balancer.py [hunk 2]
```diff
@@ -38,6 +38,7 @@ class TestLB(IntegrationTest):
 
     def test_connections_are_only_returned_once(self):
         if "PyPy" in sys.version:
+            # Tracked in PYTHON-3011
```
Comment (ShaneHarvey) [inline test/test_load_balancer.py:41]: I meant in the skiptest message itself but this is fine too.

**Review by ShaneHarvey — APPROVED** [review]
LGTM

**Pull request merged**